proposals/2020/Add Nonquantifiable Factors.md

```diff
@@ -0,0 +1,14 @@
+# Update Annual Fuel Use Native Units
+
+## Overview
+
+This proposal is to add the `auc:NonquantifiableFactors` element to `auc:PackageOfMeasures`.
```

Comment:
Nonquantifiable factors should really be interpreted as 'factors that are highly difficult to quantify, or unquantifiable given the existing infrastructure or budget'.  This might include aspects such as:
- Improve indoor air quality
- Improve occupant comfort
- Improve occupant satisfaction
- Reduce glare
- Improve access to daylight
etc.